app/services/billable_metrics/aggregations/base_service.rb

```diff
@@ -43,6 +58,26 @@ def sanitized_name(property)
       def sanitized_field_name
         sanitized_name(billable_metric.field_name)
       end
+
+      def handle_in_advance_current_usage(total_aggregation)
+        if previous_event
+          BigDecimal(previous_event.metadata['max_aggregation']) >
+          BigDecimal(previous_event.metadata['current_aggregation'])
```

Comment:
Layout/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.